scalafmt-tests/src/test/resources/scala3/OptionalBraces_unfold.stat

```diff
@@ -5373,12 +5373,12 @@ object a:
        s"($c) "
      }
    ) +
-     this.match {
-       case MateAdvice(seq, _, _, _) =>
-         seq.desc
-       case CpAdvice(judgment, _, _) =>
-         judgment.toString
-     } + "."
+     this.match
+        case MateAdvice(seq, _, _, _) =>
```

Comment:
Why do we have 3 spaces here? It should stay the same, no?

Reply:
because significant indentation in this file is set to 3:
https://github.com/scalameta/scalafmt/blob/adf6c7941753c030ccd28b3e14fd1903edbf7850/scalafmt-tests/src/test/resources/scala3/OptionalBraces_unfold.stat#L1

Reply:
ach! Right